pandas/tseries/tests/test_period.py

```diff
@@ -1745,6 +1745,15 @@ def test_constructor_simple_new(self):
         result = idx._simple_new(idx.astype('i8'), 'p', freq=idx.freq)
         self.assertTrue(result.equals(idx))
 
+    def test_shallow_copy_empty(self):
+
+        # GH13067
+        idx = PeriodIndex([], freq='M')
```

Comment:
above doesn't check empty case. Though ``_shallow_copy`` internally uses ``_simple_new``, I meant to guarantee ``_simple_new`` API.